homeassistant/components/august/__init__.py

```diff
@@ -42,7 +42,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: AugustConfigEntry) -> bo
                 hass, entry
             )
         )
-    except ValueError as err:
+    except config_entry_oauth2_flow.ImplementationUnavailableError as err:
```

Comment:
```suggestion
    except ImplementationUnavailableError as err:
```
same as your other PR